### PR TITLE
Fix `gen_any_buffer_data_constructors` in `CurrencyFormatter`

### PR DESCRIPTION
**Pull request by younies**

### CODE REVIEW

**Comment by sffc** [comment]
I think this change is not needed but I will hand this off to @robertbastian to review, both here and in #4531

**Review by sffc — COMMENTED** [review]
Why did you re-request review? My review was that I would like @robertbastian to work with you on this.

File: components/experimental/src/dimension/mod.rs [hunk 1]
```diff
@@ -51,6 +51,13 @@ impl CurrencyFormatter {
         options: options::CurrencyFormatterOptions,
         error: DimensionError,
         #[cfg(skip)]
+        functions: [
```
Comment (robertbastian) [inline components/experimental/src/dimension/mod.rs:54]: These are the default values for this parameter, so this PR doesn't change anything.

Comment (younies) [reply components/experimental/src/dimension/mod.rs:54]: Okay, got it

**Pull request closed without merging**